docker/kernel.sls

```diff
@@ -1,10 +1,11 @@
 {% from "docker/map.jinja" import docker with context %}
 
+{%- if docker.kernel is defined and grains['os_family']|lower == 'debian' %}
 pkgrepo dependencies:
   pkg.installed:
     - name: python-apt
```

Comment:
Good fix. There is no `python-apt` package on non-Debian distros so state fails.